src/server.ts

```diff
@@ -2,11 +2,8 @@ import 'module-alias/register'
 import 'source-map-support/register'
 
 import runApp from '@/helpers/runApp'
-import runMongo from '@/helpers/mongo'
 
 void (async () => {
-  console.log('Starting mongo')
-  await runMongo()
   console.log('Mongo connected')
```

Comment:
If you don't have `runMongo` here, there is no need to console `Mongo connected` :)

Reply:
you're right. thank you!